Gemfile

```diff
@@ -115,7 +118,7 @@ unless ENV['APPLIANCE']
   end
 
   group :development, :test do
-    gem "rspec-rails",      "~>3.0"
+    gem "rspec-rails",      "~>3.5.0.pre",                 :git => "git://github.com/rspec/rspec-rails", :branch => "master"
```

Comment:
Whenever it's released ;) @matthewd is right to [use the latest prerelease](https://github.com/matthewd/manageiq/commit/9ce5945b999c3a7932c3d5d7f9214de11effe1ec) directly from the repo as it's the only way to get [the fix](https://github.com/rspec/rspec-rails/commit/80637fc9ff26a4ae7b80ce96a3aa8fcc744faea2) right now.